client/my-sites/domains/domain-search/domain-search.jsx

```diff
@@ -143,7 +143,7 @@ class DomainSearch extends Component {
 						>
 							<RegisterDomainStep
 								path={ this.props.context.path }
-								suggestion={ this.props.context.params.suggestion }
+								suggestion={ this.props.context.query.s }
```

Comment:
Not technically an issue but just noting here: elsewhere in Calypso `?s=` means "search" (e.g. users, pages, media...)